content/en/monitors/types/network.md

```diff
@@ -99,5 +99,5 @@ Create a network metric monitor by following the instructions in the [metric mon
 [5]: /monitors/configuration/#no-data
 [6]: /monitors/configuration/#auto-resolve
 [7]: /monitors/configuration/#new-group-delay
-[8]: /monitors/notify/
+[8]: /monitors/types/metric
 [9]: https://app.datadoghq.com/monitors/manage
```

Comment:
```suggestion
[8]: /monitors/notify/
[9]: https://app.datadoghq.com/monitors/manage
[10]: /monitors/types/metric
```

Reply:
If you look at the file, you'll note there are three links pointing to destination link [8]. The first two links mention the Notifications page. The third link mentions the metric monitor. My recommendation is that you add a [10] link (as shown in the suggested change), and then update the link for metric monitor to point to [10], not [8].

Also, as a general FYI, if you include a forward slash in your branch name (e.g., WerNad99/change_link), it will create a preview build of the docs site with your changes. That will help the Docs team out so we can verify updates in the Docs site. It's less of a problem for small updates such as this, but for larger updates we typically need to look at the preview build.